book/website/communication/research-objects.md

```diff
@@ -0,0 +1,51 @@
+
+A Research Object (RO) {cite:ps}`Garciasilva2019-research-objects` is a method for the identification, aggregation and exchange of scholarly information on the Web. 
+ROs allow working 'open by design' and share research outputs during the research process and not only results at the end.
+The primary goal of the RO approach is to provide a mechanism to associate or link related resources about a research investigation so that they can be shared using a single identifier [{term}`def<Identifier>`] {cite:ps}`Belhajjame2015RO`.
+
+In this chapter, we will introduce ROs, their typologies and which platform and technologies exist to create and publish them.
+
+```{figure} ../figures/research-object.jpg
+---
+name: ro-lifecyle
+alt: This image shows how research objects evolve and grow in content during the collaboration process and how new research objects can be derived from existing ones.
+---
+Research Objects allow working open by design and share during the research process and not only the research outputs at the end. _The Turing Way_ project illustration by Scriberia. Used under a CC-BY 4.0 licence. DOI: [10.5281/zenodo.3332807](https://doi.org/10.5281/zenodo.3332807).
+```
+
+(cm-ro-motivation)=
+## Background & Motivation
+
+Research objects (ROs) are living resources helping to organise and describe the inputs, materials, and methods used in a scientific experiment and obtained as a result and not only at the end when publishing the research outcomes. 
+ROs encompass research outputs created, revised and shared throughout the research lifecycle that help validate findings claimed in scholarly publications.
+In short, ROs can be seen as a "single information unit" where any research material can be shared with other scientists at discrete milestones of the investigation within and outside the project.
+
+Motivation behind RO is the need to identify and share all components such as data, source code, tools, method documentation, as well as communication materials such as presentations, videos, blogs and other tangible outcomes. 
+The entire research lifecycle can be captured, allowing the release and publication of results progressively, keeping track of versioning and change information. 
+ROs facilitate reproducibility of the scientific methods and results through access to resources, context and metadata, and reuse with the possibility of forking existing ROs. 
+
+There are three guiding principles for RO:
+- Digital identity - Using unique identifiers, such as DOIs for tangible outcomes such as publications or data, and ORCID ids for researchers (explain in details in the next lesson).
+- Data aggregation - Using a method to aggregate all outcomes so that they are discoverable and hence allow anyone to investigate and reproduce the research.
+- Annotation - Use rich machine-readable metadata (discussed in open data) that help ensure findability and accessibility of all scientific work.
```

Comment:
Which section in open data is that? A link/ref to the section is helpful!

Reply:
It may have changed but I think the best reference for metadata is https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-metadata.html?highlight=metadata#metadata

Reply:
@EstherPlomp  I've added the crossreference to the Documentation and Metadata chapter as suggested by @annefou. See commit da2c7a32e7548c48f822f2f17598444e56f3bbb7